lib/src/collect.dart

```diff
@@ -73,12 +73,13 @@ Future<Map<String, dynamic>> collect(Uri serviceUri, bool resume,
 }
 
 Future<Map<String, dynamic>> _getAllCoverage(
-    VmService service, bool includeDart, Set<String> scopedOutput) async {
+    VmService service, bool includeDart, Set<String> scopedOutput, Set<String> isolateIds) async {
```

Comment:
I think `isolateIds` should probably be an optional param here as well, and then when using it below, only skip an isolate if `isolateIds` is non-null and contains that isolate's ID. As written now I think you'd get a NPE if a value is not provided to `isolateIds` when calling `collect()`

Reply:
On the first point: there's only 1 place that `_getAllCoverage` is called, so it making it optional would just be unnecessarily wordy.

On the second point: fixed!